drivers/hmis_supplemental/app/controllers/hmis_supplemental/client_data_sets_controller.rb

```diff
@@ -0,0 +1,81 @@
+###
+# Copyright 2016 - 2024 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+# a "tab" on the client dashboard for viewing a data set.
+# This controller is for both client and enrollment-based data sets
+module HmisSupplemental
+  class ClientDataSetsController < ApplicationController
+    include ClientController
```

Comment:
This controller gets flagged by brakeman for having user-controlled execution. It seems to be because of this mix-in